cmd/multiFaSequenceSwap/multiFaSequenceSwap.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/fasta"
+	"log"
+)
+
+// MultiFaSubsequenceSwap swaps a series of specified regions from a bed file between two sequences in a MultiFa object
+// The foregroundName is the name of the sequence containing the regions in the bed elements;  backgroundName is the name of the sequence including the regions outside the bed elements
+// input of this function includes the multiFa file name, the index of the background and foreground sequences from the multiFa, and a bed object with the relevant
+// start and stop indices of the swap regions
+// it will return a file containing a Fasta slice (which includes several sequences), with one more sequence that the sequences in the original
+// multiFa; this extra sequence is the background sequence with relevant foreground spots swapped as dictated by the bed file.
+func MultiFaSubsequenceSwap(inFile string, swapRegionsFile string, backgroundName, foregroundName string, outFile string, outSeqName string) {
```

Comment:
Nice PR! In general function names in command shouldn't be capitalized. Capitilized functions are public and accessible in other files, which is not necessary for cmd functions.